test/test_repository.rb

```diff
@@ -55,6 +55,14 @@ def test_repo_git_attributes
     # test/*.rb linguist-language=Java
     # Rakefile linguist-generated
     # test/fixtures/* linguist-vendored=false
+    # README.md linguist-documentation=false
+    # samples/Arduino/* linguist-documentation
+    # samples/Markdown/*.md linguist-detectable=true
+    # samples/HTML/*.html linguist-detectable=false
```

Comment:
The four lines above have been added to the `.gitattributes` in the test branch but this side hasn't been update to reflect this, so I'm adding them as part of this PR.